.gitignore

```diff
@@ -74,7 +74,6 @@
 
 .vscode
 
-.env*
```

Comment:
.env could contain secret info, are you sure you want to make it public

Reply:
Need to make sure it always have empty values when commited in repository - but it's needed to be there for deployment so far

Reply:
Let's try to replace .env with config.json